internal/providers/sdkv2/sdkv2.go

```diff
@@ -71,162 +72,175 @@ func init() {
+		"databricks_schema":                          catalog.ResourceSchema().ToResource(),
+		"databricks_secret":                          secrets.ResourceSecret().ToResource(),
+		"databricks_secret_scope":                    secrets.ResourceSecretScope().ToResource(),
+		"databricks_secret_acl":                      secrets.ResourceSecretACL().ToResource(),
+		"databricks_service_principal":               scim.ResourceServicePrincipal().ToResource(),
+		"databricks_service_principal_role":          aws.ResourceServicePrincipalRole().ToResource(),
+		"databricks_service_principal_secret":        tokens.ResourceServicePrincipalSecret().ToResource(),
+		"databricks_share":                           sharing.ResourceShare().ToResource(),
+		"databricks_sql_dashboard":                   sql.ResourceSqlDashboard().ToResource(),
+		"databricks_sql_endpoint":                    sql.ResourceSqlEndpoint().ToResource(),
+		"databricks_sql_global_config":               sql.ResourceSqlGlobalConfig().ToResource(),
+		"databricks_sql_permissions":                 access.ResourceSqlPermissions().ToResource(),
+		"databricks_sql_query":                       sql.ResourceSqlQuery().ToResource(),
+		"databricks_sql_alert":                       sql.ResourceSqlAlert().ToResource(),
+		"databricks_sql_table":                       catalog.ResourceSqlTable().ToResource(),
+		"databricks_sql_visualization":               sql.ResourceSqlVisualization().ToResource(),
+		"databricks_sql_widget":                      sql.ResourceSqlWidget().ToResource(),
+		"databricks_storage_credential":              catalog.ResourceStorageCredential().ToResource(),
+		"databricks_system_schema":                   catalog.ResourceSystemSchema().ToResource(),
+		"databricks_table":                           catalog.ResourceTable().ToResource(),
+		"databricks_token":                           tokens.ResourceToken().ToResource(),
+		"databricks_user":                            scim.ResourceUser().ToResource(),
+		"databricks_user_instance_profile":           aws.ResourceUserInstanceProfile().ToResource(),
+		"databricks_user_role":                       aws.ResourceUserRole().ToResource(),
+		"databricks_vector_search_endpoint":          vectorsearch.ResourceVectorSearchEndpoint().ToResource(),
+		"databricks_vector_search_index":             vectorsearch.ResourceVectorSearchIndex().ToResource(),
+		"databricks_volume":                          catalog.ResourceVolume().ToResource(),
+		"databricks_workspace_binding":               catalog.ResourceWorkspaceBinding().ToResource(),
+		"databricks_workspace_conf":                  workspace.ResourceWorkspaceConf().ToResource(),
+		"databricks_workspace_file":                  workspace.ResourceWorkspaceFile().ToResource(),
+	}
+
+	// Remove the resources and data sources that are being migrated to plugin framework
+	for _, dataSourceToRemove := range pluginfw.GetSdkV2DataSourcesToRemove(sdkV2Fallbacks...) {
```

Comment:
We should panic if `dataSourceToRemove` is not in `dataSourceMap`. Ditto for `resourceMap`.